implementations/cpp-valijson/bowtie_valijson.cpp

```diff
@@ -54,6 +81,9 @@ int main() {
     request.Parse(line.c_str());
     string cmd = request["cmd"].GetString();
 
+    map<string, string> os_info = getOSInfo();
```

Comment:
I think you're doing this on every request, rather than just in `start`.

Reply:
Oops! right! sorry! Fixing it :)